.vscode/settings.json

```diff
@@ -1,3 +0,0 @@
-{
```

Comment:
please add these .vscode file into .gitignore, these are your local files and not related to the project, they are not supposed to be pushed.

Reply:
resolved.